apis/suite_test.go

```diff
@@ -0,0 +1,46 @@
+package apis_test
```

Comment:
I don't think we need this file. The org handler test can just use `spec.Run` like the other handler tests do.

Reply:
Yep - deleted